client/src/components/Form/FormElement.vue

```diff
@@ -168,6 +169,20 @@ const currentValue = computed({
     },
 });
 
+/** Provides formatted select options. */
```

Comment:
This belongs in a FormSelect component that can then dispatch to FormRadio | FormMultiselect | FormWhatever.